app/services/ExactTargetService.scala

```diff
@@ -226,8 +226,10 @@ class ExactTargetService(
 }
 
 object SqsClient extends LazyLogging {
-  private val sqsClient = new AmazonSQSClient(CredentialsProvider)
-  sqsClient.setRegion(Region.getRegion(Regions.EU_WEST_1))
+  private val sqsClient = AmazonSQSClient.builder
```

Comment:
if it's chained like that I'd be tempted to put the `AmazonSQSClient.builder` on the next line just to be clearer